bluesky/callbacks/best_effort.py

```diff
@@ -441,6 +441,53 @@ def clear(self):
         self._buffer = StringIO()
         self._baseline_toggle = True
 
+    def plot_prune_fifo(self, num_lines, y_signal, x_signal):
+        """
+        Find the plot with axes x_signal and y_signal.  Replot with only the last *num_lines* lines.
+
+        .. note:: This is not a bluesky plan.  Call it as a normal Python function.
```

Comment:
This note may not be relevant anymore, as the function became a method of `BestEffortCallback`.

Reply:
I agree, I'll pull it out.